nav2_controller/src/controller_server.cpp

```diff
@@ -640,21 +655,36 @@ void ControllerServer::computeAndPublishVelocity()
 
   geometry_msgs::msg::Twist twist = getThresholdedTwist(odom_sub_->getRawTwist());
 
+  geometry_msgs::msg::PoseStamped goal =
+    path_handlers_[current_path_handler_]->getTransformedGoal(pose.header.stamp);
+  // Get the [start, end) iterators under map frame to be used for control.
+  auto [closest_point,
+    pruned_plan_end] = path_handlers_[current_path_handler_]->findPlanSegment(pose);
```

Comment:
```suggestion
  auto [closest_point, pruned_plan_end] =
    path_handlers_[current_path_handler_]->findPlanSegment(pose);
```

Just my readability preference